docs/install/windows_setup.md

```diff
@@ -158,13 +158,14 @@ git clone https://github.com/apache/incubator-mxnet.git --recursive
 3. Verify that the `DCUDNN_INCLUDE` and `DCUDNN_LIBRARY` environment variables are pointing to the `include` folder and `cudnn.lib` file of your CUDA installed location, and `C:\incubator-mxnet` is the location of the source code you just cloned in the previous step.
 4. Create a build dir using the following command and go to the directory, for example:
 ```
-mkdir C:\build
-cd C:\build
+mkdir C:\incubator-mxnet\build
+cd C:\incubator-mxnet\build
 ```
 5. Compile the MXNet source code with `cmake` by using following command:
 ```
-cmake -G "Visual Studio 15 2017 Win64" -T cuda=9.2,host=x64 -DUSE_CUDA=1 -DUSE_CUDNN=1 -DUSE_NVRTC=1 -DUSE_OPENCV=1 -DUSE_OPENMP=1 -DUSE_BLAS=open -DUSE_LAPACK=1 -DUSE_DIST_KVSTORE=0 -DCUDA_ARCH_LIST=Common -DCUDA_TOOLSET=9.2 -DCUDNN_INCLUDE=C:\cuda\include -DCUDNN_LIBRARY=C:\cuda\lib\x64\cudnn.lib "C:\incubator-mxnet"
+cmake -G "Visual Studio 15 2017 Win64" -T cuda=9.2,host=x64 -DUSE_CUDA=1 -DUSE_CUDNN=1 -DUSE_NVRTC=1 -DUSE_OPENCV=1 -DUSE_OPENMP=1 -DUSE_BLAS=open -DUSE_LAPACK=1 -DUSE_DIST_KVSTORE=0 -DUSE_MKL_IF_AVAILABLE=OFF -DCUDA_ARCH_LIST=Common -DCUDA_TOOLSET=9.2 -DCUDNN_INCLUDE=C:\cuda\include -DCUDNN_LIBRARY=C:\cuda\lib\x64\cudnn.lib "C:\incubator-mxnet"
```

Comment:
What's going on here? What does turning off `MKL` have to do with setting up the Python binding? I believe the idea is that you can opportunistically use MKL or MKL-DNN. Otherwise, use OpenBLAS.

Reply:
I will remove -DUSE_MKL_IF_AVAILABLE=OFF. The user should be allowed to use OpenBLAS, MKL, or MKL-DNN